develop/dev-guide-sample-application-python-mysqlclient.md

```diff
@@ -1,292 +1,238 @@
+        cursor.execute("UPDATE player SET goods = goods + %s, coins = coins + %s WHERE id = %s", (-amount, price, player_id))
 ```
 
-Suppose that the password you set is `123456`, and the connection parameters you get from the cluster details page are the following:
+For more information, refer to [Update Data](./dev-guide-update-data.md).
 
-- Endpoint: `xxx.tidbcloud.com`
-- Port: `4000`
-- User: `2aEp24QWEDLqRFs.root`
-
-In this case, you can modify the `get_connection` as follows:
+### Delete Data
 
 ```python
-def get_connection(autocommit: bool = True) -> MySQLdb.Connection:
-    return MySQLdb.connect(
-        host="xxx.tidbcloud.com",
-        port=4000,
-        user="2aEp24QWEDLqRFs.root",
-        password="123456",
-        database="test",
-        autocommit=autocommit,
-        ssl_mode="VERIFY_IDENTITY",
-        ssl={
-            "ca": "<ca_path>"
-        }
-    )
+with get_mysqlclient_connection(autocommit=True) as conn:
+    with conn.cursor() as cur:
+        player_id = "1"
+        cursor.execute("DELETE FROM player WHERE id = %s", (player_id,))
 ```
 
-### Step 3.3 Run the code
+For more information, refer to [Update Data](./dev-guide-delete-data.md).
 
-Before running the code, use the following command to install dependencies:
 
-```bash
-pip3 install -r requirement.txt
-```
+## Useful Notes
 
-If you need to run the script multiple times, follow the [Table initialization](#step-31-initialize-table) section to initialize the table again before each run.
+- For complete code and how to run it, see the [tidb-example-python GitHub repository](https://github.com/pingcap-inc/tidb-example-python/tree/main).
+- This Python driver is relatively low-level, so you will see a lot of SQL statements in the sample app. Unlike ORMs, there is no data object, and `mysqlclient` represents query objects with tuples. Although Python's driver is more convenient to use than those in other languages, due to its exposure to underlying implementations and the manual transaction management required, it is still recommended to use ORMs for programming unless there is a significant need for SQL. This can reduce the coupling of your app.
+- For more on how to use `mysqlclient`, refer to the [mysqlclient official documentation](https://mysqlclient.readthedocs.io/).
 
-```bash
-python3 mysqlclient_example.py
-```
 
-## Step 4. Expected output
+## Next Steps
 
-[mysqlclient Expected Output](https://github.com/pingcap-inc/tidb-example-python/blob/main/Expected-Output.md#mysqlclient)
\ No newline at end of file
+- You can continue reading the developer documentation to get more knowledge about TiDB development, such as: [Insert Data](./dev-guide-insert-data.md), [Update Data](./dev-guide-update-data.md), [Delete Data](./dev-guide-delete-data.md), [Single Table Reading](./dev-guide-get-data-from-single-table.md), [Transactions](./dev-guide-transaction-overview.md), [SQL Performance Optimization](./dev-guide-optimize-sql-overview.md), etc.
```

Comment:
We can not use the relative link like `./dev-guide-insert-data.md`. We need to use an absolute link like `/develop/dev-guide-insert-data.md` instead of it.